index.html

```diff
@@ -9,31 +9,458 @@
+                            fill="#091E42" />
+                        <path
+                            d="M175.976 65.3243C175.681 73.05 172.954 80.1136 168.532 85.0433C168.384 85.2641 168.532 85.5584 168.753 85.4848C178.187 83.4246 185.41 75.2574 185.999 65.3978C185.999 65.2507 185.852 65.1035 185.704 65.1035H176.271C176.123 65.1035 175.976 65.2507 175.976 65.3243Z"
+                            fill="#36B37E" />
+                        <path
+                            d="M151.728 65.1035H142.294C142.147 65.1035 142 65.2507 142 65.3978C142.589 75.2574 149.812 83.4246 159.246 85.4848C159.467 85.5584 159.688 85.2641 159.467 85.0433C154.971 80.1136 152.244 73.05 151.949 65.3243C152.023 65.2507 151.876 65.1035 151.728 65.1035Z"
+                            fill="#36B37E" />
+                        <path
+                            d="M152.023 62.675C152.318 54.9492 155.045 47.8857 159.467 42.9559C159.614 42.7352 159.467 42.4409 159.246 42.5145C149.812 44.5747 142.589 52.7419 142 62.6014C142 62.7486 142.147 62.8957 142.294 62.8957H151.728C151.876 62.8957 152.023 62.8221 152.023 62.675Z"
+                            fill="#091E42" />
+                    </g>
+                    <defs>
+                        <filter id="filter0_d_48_1417" x="100" y="16" width="128" height="128"
+                            filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+                            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+                            <feColorMatrix in="SourceAlpha" type="matrix"
+                                values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha" />
+                            <feOffset dy="16" />
+                            <feGaussianBlur stdDeviation="12" />
+                            <feColorMatrix type="matrix"
+                                values="0 0 0 0 0.145573 0 0 0 0 0.185183 0 0 0 0 0.5375 0 0 0 0.08 0" />
+                            <feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_48_1417" />
+                            <feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow_48_1417" result="shape" />
+                        </filter>
+                        <clipPath id="clip0_48_1417">
+                            <rect width="44" height="44" fill="white" transform="translate(142 42)" />
+                        </clipPath>
+                    </defs>
+                </svg>
+                <h1>Languages</h1>
+                <div class="tech">
+                    <div class="about-me-tag">HTML</div>
+                    <div class="about-me-tag">CSS</div>
+                </div>
```

Comment:
- [ ] Consider wrapping the `Languages, Frameworks and Skills`  in a `ul` tag and the list of technologies (e.g., "JavasScript", "HTML" and "CSS") in a `li` tag to improve code readability.